flink-cdc-common/src/main/java/com/ververica/cdc/common/event/DataChangeEvent.java

```diff
@@ -131,4 +131,20 @@ public static DataChangeEvent replaceEvent(
             TableId tableId, RecordData after, Map<String, String> meta) {
         return new DataChangeEvent(tableId, null, after, OperationType.REPLACE, meta);
     }
+
+    @Override
+    public String toString() {
+        return "DataChangeEvent{"
+                + "tableId="
+                + tableId
+                + ", before="
+                + before
+                + ", after="
+                + after
+                + ", op="
+                + op
+                + ", meta="
+                + meta
```

Comment:
the meta is a map, we could add a `String describeMeta()` utils method to visit its detail